src/inverseRobot.js

```diff
@@ -24,7 +24,24 @@
  * @return {object}
  */
 function inverseRobot(robot) {
-  // write code here
+  const returnRob = {};
+  const arrForCheckDulicate = [];
+
+  for (const key in robot) {
+    arrForCheckDulicate.push(robot[key]);
+  }
+
+  const checkIfDuplicate = (w) => new Set(w).size !== w.length;
+
+  if (checkIfDuplicate(arrForCheckDulicate)) {
+    return null;
+  }
+
+  for (const key in robot) {
+    returnRob[robot[key]] = key;
+  }
```

Comment:
Can you check: new robot already has a key or not? If you try this implement, you won’t soo many manipulations to check duplication